src/elements/Promise.tsx

```diff
@@ -0,0 +1,18 @@
+/* @jsx MD */
+import MD, { Component, PropsWithChildren } from "..";
+import { MarkdownPromise } from "../model";
+
+/**
+ * Renders a single promise that resolves to a group of tags to markdown asynchronously.
+ * Throws an error if multiple children is provided.
+ * @example
+ *   ```js
+ *   render(<Promise>{new Promise((res) => res("Hello"))}</Promise>)
+ *   ===
+ *   `Hello`
+ *   ```
+ */
+export const MdPromise: Component<PropsWithChildren<
```

Comment:
Good point that `Promise` is already taken. How about we call it `Await`? Then we could write something like `<Await>{promise}</Await>`

Reply:
Noted, will update PR will the change.